bundle/manifests/sandboxed-containers-operator.clusterserviceversion.yaml

```diff
@@ -27,46 +32,72 @@ spec:
+
+    # Requirements
+    Your cluster must be installed on bare metal infrastructure with Red Hat Enterprise Linux CoreOS workers.
+
+    # Features & benefits
+    - **Isolated Developer Environments & Privileges Scoping**
+      As a developer working on debugging an application using state-of-the-art
+      tooling you might need elevated privileges such as `CAP_ADMIN` or `CAP_BPF`. With
+      OpenShift sandboxed containers, any impact will be limited to a separate
+      dedicated kernel.
+
+    - **Legacy Containerized Workload Isolation**
+      You are mid-way in converting a containerized monolith into cloud-native
+      microservices. However, the monolith still runs on your cluster unpatched and
+      unmaintained. OpenShift sandboxed containers helps isolate it in its own kernel
+      to reduce risk.
+
+    - **Safe Multi-tenancy & Resource Sharing (CI/CD Jobs, CNFs, ..)**
+      If you are providing a service to multiple tenants, it could mean that the
+      service workloads are sharing the same resources (e.g., worker node). By
+      deploying in a dedicated kernel, the impact of these workloads have on one
+      another is greatly reduced.
+
+    - **Additional Isolation with Native Kubernetes User Experience**
+      OpenShift sandboxed containers is used as a compliant OCI runtime.
+      Therefore, many operational patterns used with normal containers are still
+      preserved including but not limited to image scanning, GitOps, Imagestreams,
+      and so on.
+
+    # How to install
+      Read the information about the Operator and click Install.
+
+      On the Install Operator page:
+
+      - Select `preview-1.2` from the list of available Update Channel options.
+      This ensures that you install the latest version of OpenShift sandboxed containers
+      that is compatible with your OpenShift Container Platform version.
+
+      - For Installed Namespace, ensure that the Operator recommended namespace
+        option is selected. This installs the Operator in the mandatory
+        `openshift-sandboxed-containers-operator` namespace, which is automatically
+        created if it does not exist. Attempting to install the OpenShift
+        sandboxed containers Operator in a namespace other than
+        `openshift-sandboxed-containers-operator` causes the installation to fail.
+
+      - For Approval Strategy, ensure that Automatic, which is the default value,
+        is selected. OpenShift sandboxed containers automatically updates when a new
+        z-stream release is available.
+
+      - Click Install to make the Operator available to the OpenShift sandboxed
+        containers namespace.
+
+      - The OpenShift sandboxed containers Operator is now installed on your
+        cluster. You can trigger the Operator by enabling the runtime on your cluster.
+        You can do this by creating a `KataConfig` CRD instance. For this click
```

Comment:
If the chance is any non-zero value that a member of the target audience will not know what a CRD is, I'd suggest that we avoid using this.  This goes for any other unexplained TLAs for that matter in my opinion (God I hate Kubernetes/OpenShift acronym soups with a vengeance! ;-)) but here it seems particularly important because if a customer doesn't know what a CRD is in the first place, they probably will have problems figuring out how to make an instance of that thing.

Reply:
@pmores What is a TLA?

/me ducks 😄 

I agree with the sentiment, even if CRD is probably one of the most well-known three letter acronyms for AOU(*)

(*) AOU: Average OpenShift Users

Reply:
I guess TLA == Three Letter Acronym ;)

Reply:
;)

Reply:
Added full form